README.md

```diff
@@ -1 +1,53 @@
-# osqp-matlab-cmake-buildsystem
\ No newline at end of file
+# osqp-matlab-cmake-buildsystem
+
+## Overview
+
+This repo contains a drop-in CMake buildsystem for the osqp-matlab bindings mantained at https://github.com/oxfordcontrol/osqp-matlab .
+
+**If you are just interested in using OSQP in MATLAB, please follow the official instructions at https://osqp.org/docs/get_started/matlab.html.**
+
+This repo is only useful if for any need to compile the MATLAB extension against a OSQP library that is already in your system, for example
+if you are also using OSQP in other software that is loaded by MATLAB, and you want to avoid ABI incompatibilities.
+
+**At the moment, this CMake build system does not support the osqp codegen functionalities. If you need that, please use the official binaries.**
+
+## Usage
+
+How to use this repo
+
+1. Install osqp in your system, and make sure that it can be find by CMake.
+
+2. Clone the repository
+
+~~~
+git clone git://github.com/dic-iit/osqp-matlab-cmake-buildsystem
```

Comment:
Shouldn't this be `git@github.com:dic-iit/osqp-matlab-cmake-buildsystem.git` or `https://github.com/dic-iit/osqp-matlab-cmake-buildsystem`? seems a mix of both :)

Reply:
It seems to work fine, no?

Reply:
Anyhow, I changed it in https://github.com/dic-iit/osqp-matlab-cmake-buildsystem/pull/1/commits/17b1aca369cbb37cb2e195440553993e5b409d87 to the URL that github suggest when you click on "Clone this repo" .

Reply:
not for me, but it's probably my problem as I realized I don't have a key, I always use https.